src/lib/rrManager.js

```diff
@@ -0,0 +1,1004 @@
+/*
+  * Licensed Materials - Property of IBM
+  * (C) Copyright IBM Corp. 2016. All Rights Reserved.
+  * US Government Users Restricted Rights - Use, duplication or
+  * disclosure restricted by GSA ADP Schedule Contract with IBM Corp.
+  */
+'use strict';
+
+const path = require('path');
+const TAG = path.basename(__filename);
+const watson = require('watson-developer-cloud');
+const csvParse = require('csv-parse');
+const DBManager = require('./dbManager');
+const env = require('./env');
+const rrDb = new DBManager({localDbName: 'rr', remoteDbName: env.cloudantDb});
```

Comment:
Use `db_rr_local` and `db_rr_remote`.  In this case env.cloudantDb is what we were using for NLC, so it will cause us to merge the NLC and RR databases when replicating to the remote DB.